SensioLabs/Security/Command/SecurityCheckerCommand.php

```diff
@@ -88,64 +84,22 @@ protected function execute(InputInterface $input, OutputInterface $output)
             return 1;
         }
 
-         if ('json' === $input->getOption('format')) {
-             $output->write(json_encode($vulnerabilities, JSON_PRETTY_PRINT));
-         } else {
-             $this->displayResults($output, $input->getArgument('lock'), $vulnerabilities);
-         }
-
-        if ($this->checker->getLastVulnerabilityCount() > 0) {
-            return 1;
+        switch ($input->getOption('format')) {
+            case 'json':
+                $formatter = new \SensioLabs\Security\Formatters\JsonFormatter();
```

Comment:
Can you move the FQDN as a use statement at the top of the file and only use the short class names here?